src/redux/reducers/index.js

```diff
@@ -1,8 +1,14 @@
 import { combineReducers } from 'redux';
 import data from 'redux/reducers/episodesReducer';
+import getCurrentPage from 'redux/reducers/paginationReducer';
 
 const rootReducer = combineReducers({
-  data
+  data,
+  getCurrentPage,
+  episodesPerPage: 12,
```

Comment:
it is still for episodes so should be inside episodesReducer
reducer for pagination shouldn't be named same as action, you can rename it into `pagination` for example or just `currentPage` key